marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubCommonTest.java

```diff
@@ -34,4 +38,39 @@ void documentPrivilegeInclusions() {
             "since it does not involve writing data");
     }
 
+    @Test
+    void verifyStatusPrivilege() {
+        runAsTestUserWithRoles(getRoleName());
+
+        DatabaseClient client = getHubClient().getFinalClient();
+        ObjectNode serverStatus = readJsonObject(client.newServerEval().javascript("xdmp.serverStatus(xdmp.host(), xdmp.server())").evalAs(String.class));
+        assertEquals(getHubClient().getDbName(DatabaseKind.FINAL), serverStatus.get("serverName").asText(),
+            "Just verifying that the user can get server status via the 'status' privilege");
+
+        ObjectNode hostStatus = readJsonObject(client.newServerEval().javascript("xdmp.hostStatus(xdmp.host())").evalAs(String.class));
+        assertNotNull(hostStatus, "Verifying user can get host status via the 'status' privilege");
+    }
+
+    @Test
+    void verifyCancelMyRequests() {
+        runAsTestUserWithRoles(getRoleName());
+
+        DatabaseClient client = getHubClient().getFinalClient();
+        try {
+            client.newServerEval().javascript("xdmp.requestCancel(xdmp.host(), xdmp.server(), xdmp.request())").evalAs(String.class);
+            fail("Expected an error because the user canceled the request");
+        } catch (FailedRequestException ex) {
+            assertTrue(ex.getServerMessage().startsWith("XDMP-CANCELED: "),
+                "Expected the call to requestCancel to succeed since the user has the cancel-my-requests privilege; message: " + ex.getServerMessage());
+        }
+    }
+
+    @Test
+    void verifySetMyRequestLimit() {
+        runAsTestUserWithRoles(getRoleName());
+
+        getHubClient().getFinalClient().newServerEval()
+            .javascript("xdmp.setRequestTimeLimit(10, xdmp.host(), xdmp.server(), xdmp.request())").evalAs(String.class);
+        logger.info("Since no error was thrown, we know that the user must have the set-my-time-limit privilege");
+    }
```

Comment:
We can use `assertDoesNotThrow` here
```suggestion
        assertDoesNotThrow(() -> getHubClient().getFinalClient().newServerEval()
                .javascript("xdmp.setRequestTimeLimit(10, xdmp.host(), xdmp.server(), xdmp.request())").evalAs(String.class),
                "Since no error was thrown, we know that the user must have the set-my-time-limit privilege");
```